decompressed/gui_file/www/lang/it-it/webui-core.po

```diff
@@ -2636,28 +2636,28 @@ msgid "Clone the content of the running bank to the inactive one. Warning: this
 msgstr "Clona il contenuto del banco attivo in quello non attivo.<br/>Attenzione: questo comporterà l'eliminazione dei dati e delle configurazioni nel banco non attivo!"
 
 msgid "This will install aria2 and ariang webui"
-msgstr "Installa Aria2 e la webUI ariang"
+msgstr "Installa Aria2 e webUI ariang"
 
 msgid "This will install the XUPnP application"
-msgstr "Installa l'applicazione XUPnP"
+msgstr "Installa applicazione XUPnP"
 
 msgid "This will install the blacklist application"
-msgstr "Installa l'applicazione Blacklist"
+msgstr "Installa applicazione Blacklist"
 
 msgid "This will install transmission and transmission-webui"
-msgstr "Installa transmission e la relativa webui"
+msgstr "Installa transmission e relativa webui"
 
 msgid "This will install uhttpd and luci"
 msgstr "Installa uhttpd e luci"
 
 msgid "This will put your gateway into LAN Update"
-msgstr "Avvia l´aggiornamento del modem"
+msgstr "Verrà avviato l'aggiornamento del modem"
 
 msgid "This will transfer the active firmware and config to the bank 1"
-msgstr "Trasferisce il firmware attivo e la configurazione nel banco 1"
+msgstr "Verrò trasferito il firmware attivo e la configurazione nel banco 1"
```

Comment:
Verrà non verrò

Reply:
@bovirus

Reply:
Già corretto nella successiva PR.

Reply:
tradurre solo la parte in italiano lascia "indietro" sia la parte inglese che tutte le altre lingue, quindi se ci si e' accorti di un errore sulla lingua inglese e' bene intervenire in primis su questa e' solo dopo su quella italiana.

letta in altri termini, sono dell'idea che se c'e un errore nella lingua inglese e' giusto che ce lo teniamo anche in italiano fin quando non risolto sulla lingua inglese

Reply:
@FrancYescO 

Direi di anadare avanti correggiamo l'Italiano (avccetando il PR) e prendiamo nota e se possibile chi inetrviene sul codice lo modifica nell'inglese.
In qeusto modo lasciamo un solo errore da corregere (nell'inglese).
Così com'è ora abbiamo due errori (in inglese e consegeuentemente in italiano).

Per favore approva la PR e la sistemiamo in un secondo step.